src/Tizen.NUI.Wearable/src/public/CircularSlider.cs

```diff
@@ -0,0 +1,674 @@
+using System.ComponentModel;
+
+namespace Tizen.NUI.Wearable
+{
+    /// <summary>
+    /// The CircularSlider class of Wearable is used to let users select a value from a continuous or discrete range of values by moving the slider thumb.
+    /// CircularSlider shows the current value with the length of the line.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class CircularSlider : Control
+    {
+        #region Fields
+
+        /// <summary>Bindable property of Thickness</summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty ThicknessProperty = BindableProperty.Create(nameof(Thickness), typeof(float), typeof(CircularSlider), default(float), propertyChanged: (bindable, oldValue, newValue) =>
+        {
+            var instance = ((CircularSlider)bindable);
+
+            // TODO Set viewStyle.Thickness after style refactoring done.
+
+            instance.UpdateVisualThickness((float)newValue);
+        },
+        defaultValueCreator: (bindable) =>
+        {
+            return ((CircularSliderStyle)((CircularSlider)bindable).viewStyle)?.Thickness;
+        });
+
+        /// <summary>Bindable property of MaxValue</summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty MaxValueProperty = BindableProperty.Create(nameof(MaxValue), typeof(float), typeof(CircularSlider), default(float), propertyChanged: (bindable, oldValue, newValue) =>
+        {
+            var instance = (CircularSlider)bindable;
+            if (newValue != null)
+            {
+                instance.maxValue = (float)newValue;
+                instance.UpdateValue();
+            }
+        },
+        defaultValueCreator: (bindable) =>
+        {
+            var instance = (CircularSlider)bindable;
+            return instance.maxValue;
+        });
+
+        /// <summary>Bindable property of MinValue</summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty MinValueProperty = BindableProperty.Create(nameof(MinValue), typeof(float), typeof(CircularSlider), default(float), propertyChanged: (bindable, oldValue, newValue) =>
+        {
+            var instance = (CircularSlider)bindable;
+            if (newValue != null)
+            {
+                instance.minValue = (float)newValue;
+                instance.UpdateValue();
+            }
+        },
+        defaultValueCreator: (bindable) =>
+        {
+            var instance = (CircularSlider)bindable;
+            return instance.minValue;
+        });
+
+        /// <summary>Bindable property of CurrentValue</summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty CurrentValueProperty = BindableProperty.Create("currentValue", typeof(float), typeof(CircularSlider), default(float), propertyChanged: (bindable, oldValue, newValue) =>
```

Comment:
Please use `nameof` instead of string(StyleCop rule violation(CA1507)).
https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1507?view=vs-2019